src/main/java/net/fabricmc/tinyremapper/TinyRemapper.java

```diff
@@ -1142,17 +1178,15 @@ public void run() {
 
 				if (propagateRecordComponents != LinkedMethodPropagation.DISABLED
 						&& cls.isRecord()
-						&& type == MemberType.FIELD
+						&& type == MemberHeader.MemberType.FIELD
 						&& (member.access & (Opcodes.ACC_STATIC | Opcodes.ACC_PRIVATE | Opcodes.ACC_FINAL)) == (Opcodes.ACC_PRIVATE | Opcodes.ACC_FINAL)) { // not static, but private+final
 					String getterIdSrc = MemberInstance.getMethodId(member.name, "()".concat(member.desc));
-					MemberInstance getter = cls.getMember(MemberType.METHOD, getterIdSrc);
+					MemberInstance getter = cls.getMember(MemberHeader.MemberType.METHOD, getterIdSrc);
 
 					if (getter != null && getter.isVirtual()) {
 						visitedUp.add(cls);
 						visitedDown.add(cls);
-						cls.propagate(TinyRemapper.this, MemberType.METHOD, className, getterIdSrc, nameDst,
-								Direction.ANY, true, true,
-								true, visitedUp, visitedDown);
+						cls.propagate(TinyRemapper.this, MemberHeader.MemberType.METHOD, className, getterIdSrc, nameDst, Direction.ANY, true, true, true, visitedUp, visitedDown);
```

Comment:
Please avoid unrelated changes, the MemberType changes above can be eliminated by changing the import as well.

Reply:
this fails checkstyle